tests/test_estimation.py

```diff
@@ -10,6 +10,9 @@
 
 cdnow_customers = load_cdnow()
 cdnow_customers_with_monetary_value = load_summary_data_with_monetary_value()
+returning_cdnow_customers_with_monetary_value = cdnow_customers_with_monetary_value[
+    cdnow_customers_with_monetary_value['frequency'] > 0
+]
```

Comment:
yea I think it's better to move this into the test, to make it explicit. I wonder why there are zero money values in the source data 😕